docs/resources/dns_split_nameservers.md

```diff
@@ -0,0 +1,33 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "tailscale_dns_split_nameservers Resource - terraform-provider-tailscale"
+subcategory: ""
+description: |-
+  The dnssplitnameservers resource allows you to configure split DNS nameservers for your Tailscale network. See https://tailscale.com/kb/1054/dns for more information.
```

Comment:
Tried to snake case this to match the actual resource name but the autogen tool / CI didn't like that so 🤷